### DISCUSSION TITLE
Welcome to dlt-daemon Discussions!

### DISCUSSION THREAD

**Discussion by michael-methner** [COVESA/dlt-daemon#430] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by mimm83** [comment]
Hi,
Just a question regarding usage.
For example if I would like to start a logging with dlt-receive and then change the log level for just one or two application. How should the procedure looks like ?

Given an example:
1. Start logging, example : dlt-receive -o ecu_dlt_log.dlt -c 100M 169.252.4.12 -p 3591
2. Change log level to a specific APP, for instance APP1 logging in debug mode : dlt-control -a APP1 -l 5 169.254.4.12
Is it correct?

Thank you in advance!

**Reply by minminlittleshrimp** [reply]
Hello @mimm83 ,
thanks for your interest in dlt.

It is not clear to me the case you mention. If your are concerning about dlt behaviors when sending a control message for changing the log level while logging, then I would say that we have our own mechanism in dlt-daemon to reset the log level. Hence, after changing log level, expected logs will be received with dlt-receive.
Hello @michael-methner @lvklevankhanh , please kindly correct me if I am wrong.
Regards

**Comment by mimm83** [comment]
it's solved. the command I mentioned in the point 2 was not working for because I was using  an older version.

**Comment by NivedhaChandrabose** [comment]
Hi,
How to check the drop rate of logs in Covesa DLT. Are there any API available for the calculation of drop rate?

**Reply by minminlittleshrimp** [reply]
Hello @NivedhaChandrabose 

I think you could have a look at [DLT KPI](https://github.com/COVESA/dlt-daemon/blob/master/doc/dlt_kpi.md)

I am not used to dlt kpi but I think CHK message can inform you the lost logs number and may somehow help the case I think.

Regards
